.github/workflows/main.yaml

```diff
@@ -306,7 +308,11 @@ jobs:
         uses: ./.github/actions/setup-go
 
       - name: Clone Lit repository
-        run: git clone https://github.com/lightninglabs/lightning-terminal.git
+        uses: actions/checkout@v3
+        with:
+          repository: lightninglabs/lightning-terminal
+          ref: ${{ env.LITD_ITEST_BRANCH }}
```

Comment:
not sure if related, but there's a tiny misbehavior according to an itest
https://github.com/lightninglabs/taproot-assets/actions/runs/10715230733/job/29710290030?pr=1116#step:4:52159

we get a "status completed" instead of "proofs transferred"

Reply:
Hmm, that seems to be a GitHub only flake... I ran this test a bunch of time locally and it never failed for me...
Going to re-run CI and just hope it goes away.

Reply:
Man, this has been driving me bananas! I had something similar way back and never understood why that happens on GitHub.

The weird thing is that in my case when the test ran on Github it expected a different state than what the test was actually supposed to be expecting, and I think that's the case hear as well, right?

The test expects `SendStateWaitTxConf`
https://github.com/lightninglabs/taproot-assets/blob/6f4d65fa7ad1c0b7e40b543467c4dea0ebbde9aa/itest/psbt_test.go#L1148-L1152

But according to the run on GitHub the test expects `SendStateTransferProofs` which makes no sense whatsoever:
https://github.com/lightninglabs/taproot-assets/actions/runs/10715230733/job/29710290030?pr=1116#step:4:52158

Maybe an enumeration is off by one?

Reply:
The enum values passed into `AssertSendEvents()` is a "from" and "to". So it's expecting all events from `SendStateWaitTxConf` in order up until `SendStateComplete`. And `SendStateTransferProofs` just happens to be an in-between one.
So I think it's a pure timing issue, perhaps because GitHub is slower than the local machine.

Reply:
Oh, I misunderstood that. Thanks!